lib/IDE/CodeCompletion.cpp

```diff
@@ -1396,12 +1396,23 @@ bool CodeCompletionCallbacksImpl::trySolverCompletion(bool MaybeFuncBody) {
                           CurDeclContext, CompletionContext, Consumer);
     return true;
   }
-  case CompletionKind::StmtOrExpr:
+  case CompletionKind::AccessorBeginning:
   case CompletionKind::ForEachSequence:
-  case CompletionKind::PostfixExprBeginning: {
-    assert(CodeCompleteTokenExpr);
+  case CompletionKind::PostfixExprBeginning:
+  case CompletionKind::StmtOrExpr: {
     assert(CurDeclContext);
 
+    addKeywords(CompletionContext.getResultSink(), MaybeFuncBody);
+
+    if (!CodeCompleteTokenExpr) {
+      // Happens when completing inside e.g.
+      //   var x: Int {
+      //     get { ... }
+      //     #^COMPLETE^#
+      //   }
```

Comment:
I'm confused. You modified `parseGetSet` to pass the `CodeCompletionExpr`. Is this reachable?

Reply:
If we don't have a test case checking _not_ suggesting expressions here, could you add some?

Reply:
Good catch, I did apparently fix the bug multiple times in incompatible ways. I updated the PR.

We basically have test cases to test we aren’t producing expressions in `complete_accessor.swift`, where a lot of global results are disallowed in these places. The way it worked in this PR was that completion lookup simply failed completely and didn’t produce any results, which left us with the keyword results only.